spring-session-samples/spring-session-sample-boot-mongodb-reactive/spring-session-sample-boot-mongodb-reactive.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'io.spring.convention.spring-sample-boot'
+
+dependencies {
+	compile project(':spring-session-data-mongodb')
+	compile "org.springframework.boot:spring-boot-starter-webflux"
+	compile "org.springframework.boot:spring-boot-starter-thymeleaf"
+	compile "org.springframework.boot:spring-boot-starter-data-mongodb-reactive"
+    compile "de.flapdoodle.embed:de.flapdoodle.embed.mongo"
```

Comment:
I added an embedded MongoDB to both samples. 
The alternative is to expect the user to have a local MongoDB instance on their machine before running the sample.  In that case we would need to update the integration tests for the samples to use test containers.